oras/auth/base.py

```diff
@@ -19,6 +19,7 @@ class AuthBackend:
 
     def __init__(self, *args, **kwargs):
         self._auths: dict = {}
+        self.prefix: str = "https"
```

Comment:
step 4.

the base `AuthBackend` by default would have a property `prefix` valorized to https, but as we just seen in step 3, this is set to the actual prefix value.